shell/browser/api/atom_api_browser_view.cc

```diff
@@ -98,12 +98,12 @@ void BrowserView::WebContentsDestroyed() {
 // static
 mate::WrappableBase* BrowserView::New(mate::Arguments* args) {
   if (!Browser::Get()->is_ready()) {
-    args->ThrowError("Cannot create BrowserView before app is ready");
+    args->ThrowError("BrowserView can only be created after app is ready.");
     return nullptr;
   }
 
   if (args->Length() > 1) {
-    args->ThrowError("Too many arguments");
+    args->ThrowError("Too many arguments specified.");
```

Comment:
Why is this even an error? Can't we just ignore the extra arguments 🤔 That's how JS functions / classes work

Reply:
Good question, i agree and will remove.

Reply:
Addressed with https://github.com/electron/electron/pull/19375!